### DISCUSSION TITLE
Filtering on multi-relation with "and"-conjunction

### DISCUSSION THREAD

**Discussion by FlixMa** [pocketbase/pocketbase#4507] [Q&A] [answered]
Hey all, hey @ganigeorgiev,
Based on your [examplary db structure](https://pocketbase.io/docs/working-with-relations/), how would I fetch all the `posts` that are tagged with a given set of `tags` (and-conjunction)?

So lets say I have three tags a, b and c and I have two posts: 
1. `x` tagged with `a`
2. `y` tagged with `b` and `c`

I tried the following syntax `tags.name ?= "a"` which works fine (returns `x`), but as soon as I require two or more tags `(tags.name ?= "b") && (tags.name ?= "c")` it does not return anything (`y` expected). I recon in previous versions `("b" ~ tags.name) && ("c" ~ tags.name)` might have worked, but that's not the case (anymore?).

Could you give me hint?
Thanks

**Comment by ganigeorgiev** [comment, marked as answer]
_@FlixMa Please don't ping me for no reason. I cannot provide and you shouldn't expect dedicated support._

Using `?=` will not work in this case as the `tags.name` row value cannot be 2 different things at the same time. It is explained in more details why this is the case in https://github.com/pocketbase/pocketbase/discussions/4319.

Using the `~` operator should work but you need to reverse the arguments and search by the ids in the raw serialized array value (_see the example in the above discussion_).

**Reply by FlixMa** [reply]
Im sorry, I didn’t want to bother you. It wasn’t my intention to make you aware of this as soon as possible, just wanted to say hi. I appreciate your time and effort.
